flow/shared/alerting/alerting.go

```diff
@@ -55,9 +58,20 @@ func NewAlerter(catalogPool *pgxpool.Pool) (*Alerter, error) {
 	if catalogPool == nil {
 		return nil, errors.New("catalog pool is nil for Alerter")
 	}
-
+	snsTopic := os.Getenv("TELEMETRY_AWS_SNS_TOPIC_ARN")
+	var snsMessageSender telemetry.Sender
+	if snsTopic != "" {
+		var err error
+		snsMessageSender, err = telemetry.NewSNSMessageSenderWithNewClient(context.TODO(), &telemetry.SNSMessageSenderConfig{
```

Comment:
Is this the right context to use over here? Why this vs `context.Background` ?

Reply:
was not sure which context to pull as the method did not accept one yet. can shift to `context.Background`

Reply:
Function should be updated to take a context instead